src/Config.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Tripod;
+
+class Config implements ITripodConfig
+{
+    /**
+     * @var Config
+     */
+    private static $instance;
+
+    /**
+     * @var array
+     */
+    private static $config = [];
+
+    /**
+     * Config should not be instantiated directly: use Config::getInstance()
+     */
+    private function __construct()
+    {
```

Comment:
Not sure what you mean here - this is private because it's a singleton.

You would call:
```
\Tripod\Config::setConfig([...some config...]);
$configInstance = \Tripod\Config::getInstance();
```

Just like we currently do with `\Tripod\Mongo\Config`